bigchaindb/web/views/transactions.py

```diff
@@ -35,74 +36,83 @@ def record(state):
                          'performance.')
 
 
-@transaction_views.route('/transactions/<tx_id>')
-def get_transaction(tx_id):
-    """API endpoint to get details about a transaction.
+class TransactionApi(Resource):
+    def get(self, tx_id):
+        """API endpoint to get details about a transaction.
 
-    Args:
-        tx_id (str): the id of the transaction.
+        Args:
+            tx_id (str): the id of the transaction.
 
-    Return:
-        A JSON string containing the data about the transaction.
-    """
+        Return:
+            A JSON string containing the data about the transaction.
+        """
+        pool = current_app.config['bigchain_pool']
 
-    pool = current_app.config['bigchain_pool']
+        with pool() as bigchain:
+            tx = bigchain.get_transaction(tx_id)
 
-    with pool() as bigchain:
-        tx = bigchain.get_transaction(tx_id)
+        if not tx:
+            return make_error(404)
 
-    if not tx:
-        return make_error(404)
+        return tx
 
-    return flask.jsonify(**tx)
 
+class TransactionStatusApi(Resource):
+    def get(self, tx_id):
+        """API endpoint to get details about the status of a transaction.
 
-@transaction_views.route('/transactions/', methods=['POST'])
-def create_transaction():
-    """API endpoint to push transactions to the Federation.
+        Args:
+            tx_id (str): the id of the transaction.
 
-    Return:
-        A JSON string containing the data about the transaction.
-    """
-    pool = current_app.config['bigchain_pool']
-    monitor = current_app.config['monitor']
+        Return:
+            A JSON string containing the status of the transaction.
```

Comment:
Since the conversion to JSON is happening outside the method, I'd change the description to:
```
A ``dict`` in the format ``{'status': <status>}``, where ``<status>``
is one of "valid", "invalid", "undecided", "backlog".
```